README.md

```diff
@@ -25,52 +26,66 @@ This section lists all quantum-safe algorithms supported by this oqs testbed.
 
-### Code points/algorithm IDs
-
 <!--- OQS_TEMPLATE_FRAGMENT_IDS_START -->
-|Algorithm name | default ID | enabled | environment variable |
-|---------------|:----------:|:-------:|----------------------|
-| frodo640aes | 0x0200 | Yes | OQS_CODEPOINT_FRODO640AES |
-| frodo640shake | 0x0201 | Yes | OQS_CODEPOINT_FRODO640SHAKE |
-| frodo976aes | 0x0202 | Yes | OQS_CODEPOINT_FRODO976AES |
-| frodo976shake | 0x0203 | Yes | OQS_CODEPOINT_FRODO976SHAKE |
-| frodo1344aes | 0x0204 | Yes | OQS_CODEPOINT_FRODO1344AES |
-| frodo1344shake | 0x0205 | Yes | OQS_CODEPOINT_FRODO1344SHAKE |
-| kyber512 | 0x023A | Yes | OQS_CODEPOINT_KYBER512 |
-| p256_kyber512 | 0x2F3A | Yes | OQS_CODEPOINT_P256_KYBER512 |
-| kyber768 | 0x023C | Yes | OQS_CODEPOINT_KYBER768 |
-| p384_kyber768 | 0x2F3C | Yes | OQS_CODEPOINT_P384_KYBER768 |
-| kyber1024 | 0x023D | Yes | OQS_CODEPOINT_KYBER1024 |
-| bikel1 | 0x0241 | Yes | OQS_CODEPOINT_BIKEL1 |
-| bikel3 | 0x0242 | Yes | OQS_CODEPOINT_BIKEL3 |
-| bikel5 | 0x0243 | Yes | OQS_CODEPOINT_BIKEL5 |
-| hqc128 | 0x022C | Yes | OQS_CODEPOINT_HQC128 |
-| hqc192 | 0x022D | Yes | OQS_CODEPOINT_HQC192 |
-| hqc256 | 0x022E | Yes | OQS_CODEPOINT_HQC256 |
-<!--- OQS_TEMPLATE_FRAGMENT_IDS_END -->
+|Algorithm name | Enabled |  Type
+|---------------|----------------------|----------------------|
+| bikel1           | Yes     | Post Quantoum |
+| bikel3           | Yes     | Post Quantoum |
+| bikel5           | Yes     | Post Quantoum |
+| frodo1344aes     | Yes     | Post Quantoum |
+| frodo1344shake   | Yes     | Post Quantoum |
+| frodo640aes      | Yes     | Post Quantoum |
+| frodo640shake    | Yes     | Post Quantoum |
+| frodo976aes      | Yes     | Post Quantoum |
+| frodo976shake    | Yes     | Post Quantoum |
+| hqc128           | Yes     | Post Quantoum |
+| hqc192           | Yes     | Post Quantoum |
+| hqc256           | Yes     | Post Quantoum |
+| kyber1024        | Yes     | Post Quantoum |
+| kyber512         | Yes     | Post Quantoum |
+| kyber768         | Yes     | Post Quantoum |
+| p256_kyber512    | Yes     | Hybrid        |
+| p384_kyber768    | Yes     | Hybrid        |
+| prime256v1       | Yes     | Classic       |
+| secp384r1        | Yes     | Classic       |
+| x25519_kyber768  | Yes     | Hybrid        |
 
+<!--- OQS_TEMPLATE_FRAGMENT_IDS_END -->
 
-# Installation & Setup
+# Getting Started
 
-Install `npm` and `node.js`. Instructions can be found [here](https://nodejs.org/en/download).
+We suggest using the docker-compose distribution via the instructions below, but there is also a [Development Installation](#development) procedure if you’d prefer to run the Qujata in development mode.
 
-Initialize `api` by the following stages:
+1. To start, clone the qujata repository.
+```bash
+git clone https://github.com/att/qujata.git
+cd qujata
 ```
-$ git clone <ENTER_GITHUB_BACKEND_URL_HERE>
-$ cd api
-$ npm install
+2. Start the application using:
```

Comment:
we used '.' to end the line on #1 and ':' on #2
let's keep a same convention for both